src/legacy/core_plugins/kibana/public/discover/get_inner_angular.ts

```diff
@@ -0,0 +1,332 @@
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// inner angular imports
+// these are necessary to bootstrap the local angular.
+// They can stay even after NP cutover
+import angular from 'angular';
+import 'ui/angular-bootstrap';
+import { IPrivate } from 'ui/private';
+import { EuiIcon } from '@elastic/eui';
+// @ts-ignore
+import { EventsProvider } from 'ui/events';
+import { PersistedState } from 'ui/persisted_state';
+// @ts-ignore
+import { PromiseServiceCreator } from 'ui/promises/promises';
+// @ts-ignore
+import { createEsService } from 'ui/es';
+import { i18nDirective, i18nFilter, I18nProvider } from '@kbn/i18n/angular';
+// @ts-ignore
+import { PrivateProvider } from 'ui/private/private';
+import { CoreStart, LegacyCoreStart, UiSettingsClientContract } from 'kibana/public';
+// @ts-ignore
+import { watchMultiDecorator } from 'ui/directives/watch_multi/watch_multi';
+// @ts-ignore
+import { registerListenEventListener } from 'ui/directives/listen/listen';
+// @ts-ignore
+import { KbnAccessibleClickProvider } from 'ui/accessibility/kbn_accessible_click';
+// @ts-ignore
+import { FieldNameDirectiveProvider } from 'ui/directives/field_name';
+// @ts-ignore
+import { CollapsibleSidebarProvider } from 'ui/collapsible_sidebar/collapsible_sidebar';
+// @ts-ignore
+import { CssTruncateProvide } from 'ui/directives/css_truncate';
+// @ts-ignore
+import { FixedScrollProvider } from 'ui/fixed_scroll';
+// @ts-ignore
+import { DebounceProviderTimeout } from 'ui/directives/debounce/debounce';
+// @ts-ignore
+import { AppStateProvider } from 'ui/state_management/app_state';
+// @ts-ignore
+import { GlobalStateProvider } from 'ui/state_management/global_state';
+// @ts-ignore
+import { createRenderCompleteDirective } from 'ui/render_complete/directive';
+// @ts-ignore
+import { StateManagementConfigProvider } from 'ui/state_management/config_provider';
+// @ts-ignore
+import { KbnUrlProvider, RedirectWhenMissingProvider } from 'ui/url';
+// @ts-ignore
+import { createTopNavDirective, createTopNavHelper } from 'ui/kbn_top_nav/kbn_top_nav';
+import { configureAppAngularModule } from 'ui/legacy_compat';
+// @ts-ignore
+
+// @ts-ignore
+import { IndexPatterns } from '../../../data/public/index_patterns/index_patterns';
```

Comment:
No need for @ts-ignore on index patterns as well as storage

Reply:
thx! done!